lib/core/ui/responsive_builder.dart

```diff
@@ -3,13 +3,11 @@ import 'package:flutter/material.dart';
 
 class ResponsiveBuilder extends StatelessWidget {
   const ResponsiveBuilder({
-    @required this.smallWidget,
-    @required this.largeWidget,
-    this.mediumWidget,
```

Comment:
Her widget'da mediumWidget kullanmadık diye hatırlıyorum.
Satır 18'de 
```
final Widget? mediumWidget;
```
şeklinde düzenleyerek

Satır 29'u geri alabiliriz diye düşünüyorum.

Reply:
We do not have a medium widget for each widget. Can we make it optional here and if it is null, we need to use the small widget.